docs/infra.md

```diff
@@ -0,0 +1,92 @@
+# Infra
+
+Infrastructure automation is provided by [Pulumi](https://www.pulumi.com/) and accessible via the `infra` subcommand.
+
+**NOTE**
+The infra command is experimental and is subject to rapid change.
+Currently only AWS and Cloudflare are supported.
+
+```bash
+$ audius-ctl infra --help
+
+Manage audius-d instances
+
+Usage:
+  audius-ctl infra [command] [flags]
+  audius-ctl infra [command]
+
+Available Commands:
+  cancel      Cancel the current in progress update
+  destroy     Destroy the current context
+  output      Get a specific output from the stack
+  update      Update (deploy) the current context
+```
+
+### Configuration
+
+Run `audius-ctl config edit`. Add the infra section to your context config.
+
+```yaml
+network:
+  deployOn: mainnet
+  infra:                             <------
+    cloudflareAPIKey: my_cf_apikey   <------
+    cloudflareZoneId: my_cf_zoneid   <------
+    cloudflareTld: example.com       <------
+    awsAccessKeyID: my_id            <------
+    awsSecretAccessKey: my_secret    <------
+    awsRegion: us-east-2             <------
+nodes:
+  audius-d-creator.example.com:
+    type: creator
+    version: prerelease
+    privateKey: 21118f9a6de181061a2abd549511105adb4877cf9026f271092e6813b7cf58ab
+    wallet: 0x0D38e653eC28bdea5A2296fD5940aaB2D0B8875c
+    rewardsWallet: 0xb3c66e682Bf9a85F6800c769AC5A05c18C3F331d
+```
+
+### Deploy
+
+To provision infrastructure for your context, run
+```bash
+audius-ctl infra update [-y]
+```
+
+To wire this up with audius-ctl (on your local machine), run the below script to add to your ssh config (defaults to `~/.ssh/config` if not specified).
+```bash
+./scripts/infra_ssh_config.sh ~/.ssh/config_personal
+
+# also make the key readonly by the owner
+chmod 0400 $(audius-ctl infra output instancePrivateKeyFilePath)
+```
+
+Now, startup the service
+```bash
+audius-ctl up
```

Comment:
This produced some error messages and prompted me about "the authenticity of the host cannot be verified blah blah" standard message when ssh'ing to a new machine, but it still completed successfully so this is not a blocker. If there's an easy way to retrieve the host fingerprint from pulumi or something so we can just preload it in ~/.ssh/known_hosts lmk